presto-docs/src/main/sphinx/connector/blackhole.rst

```diff
@@ -79,3 +82,29 @@ table property (default value is equal to 16)::
       rows_per_page = 2000,
       field_length = 100
     );
+
+The consuming/producing rate can be slowed down
+using the ``page_processing_delay`` table property.
+Setting this property to '5s' will lead to 5 seconds
+sleep before consuming or producing new page::
+
+    CREATE TABLE blackhole.test.delay (
+      dummy bigint
+    )
+    WITH (
+      split_count = 1,
+      pages_per_split = 1,
+      rows_per_page = 1,
+      page_processing_delay = '5s'
+    );
+
+    select count(*) from blackhole.test.delay;
+     _col0
+    -------
+         1
+    (1 row)
+
+    Query 20151109_104031_00006_tp8hq, FINISHED, 1 node
```

Comment:
I wanted to show that this query actually takes 5s to execute. If you think that this is extra, than i'm going to remove the result at all.